summoner-cli/src/Summoner/Config.hs

```diff
@@ -32,6 +32,7 @@ module Summoner.Config
 
 import Data.List (lookup)
 import Generic.Data (gmappend, gmempty)
+import Generic.Data.Orphans ()
```

Comment:
Do I understand this correctly, that this is needed only for older versions of GHC?

Reply:
Yes, i found it from lib author. It didn't build 8.4.4 without orphans import.